src/modal.js

```diff
@@ -1,32 +1,59 @@
-import React, { useState } from "react";
+import React, { useEffect,useState } from "react";
+import { ScrubberBar } from "./scrubber-bar-component";
 import ReactModal from "react-modal";
 
 export function VidModal({ isOpen, onClose, vidSource, setBackTime }) {
   const modalVideo = React.createRef(null);
-  const [stopPos, setStopPos] = useState(0);
-  const [startPos, setStartPos] = useState(0);
+  const [stopTime, setStopTime] = useState(0);
+  const [startTime, setStartTime] = useState(0);
+  const [recordState, setRecordState] = useState(false);
+  const [elapsedTime, setElapsedTime] = useState(0);
+  const [playState, setPlayState] = useState(false);
   const [videoDuration, setVideoDuration] = useState(0);
-  const increment = 50 / videoDuration;
   const vidTimeControls = 5;
+  const increment = 50 / videoDuration;
+
+  useEffect(() => {
+    if (modalVideo.current) {
+      setVideoDuration(modalVideo.current.duration);
+    }
+  }, [modalVideo]);
+
+  function calcPosition() {
+    setElapsedTime(modalVideo.current.currentTime * increment);
+  }
 
   function setOrigin() {
     modalVideo.current.currentTime = setBackTime;
   }
 
-  function clearPos() {
-    setStartPos(0);
-    setStopPos(0);
+  function handlePlayClick() {
+    if (modalVideo.current.paused === true) {
+      modalVideo.current.play();
+      setPlayState(true);
+    } else {
+      modalVideo.current.pause();
+      setPlayState(false);
+    }
+  }
+
+  function clearTime() {
+    setStartTime(0);
+    setStopTime(0);
+    console.log("start: "+startTime+"stop: "+stopTime);
   }
 
   function getStartTimeStamp() {
     if (!!modalVideo) {
-      setStartPos(modalVideo.current.currentTime * increment);
+      setStartTime(modalVideo.current.currentTime);
+      console.log("start time: "+ modalVideo.current.currentTime);
```

Comment:
not sure if these were left over from debugging but if they are not needed we can probably remove thme